README.md

```diff
@@ -1,13 +1,86 @@
 
-cf-plot is a set of Python plotting routines for the contour, vector and line plots that climate
-researchers commonly make.  
+## Code-light plotting for earth science and aligned research
 
+### Overview
 
-Documentation
-=============
-Please refer to the cf-plot homepage http://ajheaps.github.io/cf-plot
+**cf-plot allows you to produce and customise publication-quality contour,
+vector, line and more plots with the power of Python,
+[matplotlib](https://matplotlib.org/),
+[Cartopy](https://scitools.org.uk/cartopy/docs/latest/) and
+[cf-python](https://ncas-cms.github.io/cf-python/), in as few lines of code
+as possible.**
 
+It is designed to be a useful visualisation tool for environmental, earth
+and aligned science, for example to facilitate climate and meteorological
+research. cf-plot is developed and maintained by the
+[NCAS-CMS](https://cms.ncas.ac.uk/index.html) group, part of
+[NCAS](https://ncas.ac.uk/).
 
+
+### Brief Demonstration
+
+In as little as four lines of Python including imports, using `cf-plot` you can
+for example produce a contour plot showing a 2D subspace of a netCDF dataset:
+
+```python
+import cf
+import cfplot as cfp
+f = cf.read('<dataset name>.nc')[0]  # picks out a read-in field of the dataset
+cfp.con(f.subspace(time=<chosen time value>))  # creates a contour plot of the field at that time value
+```
+
+
+### Examples Gallery
+
+A gallery of outputs made with cf-plot, showcasing a range of plotting
+possibilities with links to relevant documentation pages and to example code,
+can be found
+[on this dedicated page within the documentation](http://ajheaps.github.io/cf-plot/gallery.html),
+as illustrated in this (static) image:
+
+![cf-plot example gallery of plots](docs/media/cf_gallery_image.png)
+
+
+### Documentation
+
+See [the cf-plot homepage](http://ajheaps.github.io/cf-plot)
+(`http://ajheaps.github.io/cf-plot`) for the full online documentation.
+
+
+### Installation
+
+To install cf-plot with its required dependencies, you can use `pip`:
+
+```bash
+pip install cf-python
+pip install cf-plot
+```
+
+or you can use `conda` (or similar package managers such
+as `mamba`) as follows (or equivalent):
+
+```bash
+conda install -c ncas -c conda-forge cf-python cf-plot udunits2
+conda install -c conda-forge mpich esmpy
+```
+
+More detail about installation is provided on the
+[installation page](http://ajheaps.github.io/cf-plot/download.html)
+(`http://ajheaps.github.io/cf-plot/download.html`)
+of the documentation.
+
+### Contributing
+
+Everyone is welcome to contribute to cf-plot, however the NCAS-CMS team reserve
+the right to only accept changes which they consider to improve the codebase
+and align with future ambitions for the library.
+
+Contributing guidelines will be added to the repository shortly.
+
+
+### Help: Issues, Questions, Feature Requests, etc.
+
+For any queries, see the
+[guidance page](http://ajheaps.github.io/cf-plot/issues.html)
+(`http://ajheaps.github.io/cf-plot/issues.html`).
```

Comment:
Surely https://github.com/NCAS-CMS/cf-plot/issues ?

Reply:
I haven't yet moved the documentation source and build across to the NCAS-CMS space, so technically that is the canonical link at present, though Andy has set up redirection for now. I was planning to update the links once I had done the move, but you're right it might be best just put the eventual ones in for now (will be less confusing to refer to the same user/org space as the repo belongs to). Thanks.

Reply:
Oh, realising I misinterpreted what you said in my reply above (ignore it).

Andy's current guidance page states people should email me directly. I have started to initiate the Issue Tracker for this instead, but haven't had a chance to update the guidance. Until I update that I will leave this pointer so as not to make contradictory statements RE issue reporting.

Reply:
Hope that makes sense and is OK to you. I'll leave this unresolved in case you want to say otherwise :slightly_smiling_face: